vaultsrv/admin.py

```diff
@@ -1,3 +1,8 @@
-# from django.contrib import admin
+from django.contrib import admin
 
-# Register your models here.
+from vaultsrv.models import (
+    Account, AccountGroup, GroupMembership, Customer, ServiceGroup, Service, ServiceGroupMembership
+)
+
+# This is no the final admin definition, it's is only to test the models
+admin.site.register((Account, AccountGroup, GroupMembership, Customer, ServiceGroup, Service, ServiceGroupMembership))
```

Comment:
Could you break these lines into two so they fall under 80 characters per line? :)